src/nodes/interceptor/eth_sign.go

```diff
@@ -19,7 +19,7 @@ func (i *Interceptor) ethSign(ctx context.Context, from ethcommon.Address, data
 		return nil, err
 	}
 
-	sig, err := store.Sign(ctx, from, data)
+	sig, err := store.SignMessage(ctx, from, data.String())
```

Comment:
This is intended to use same behaviour than geth does when user calls `eth_sign`. Looking at `go-ethereum` code base it was doing raw signing  using cryptp.ecdsa as you can see in the following code:

https://github.com/ethereum/go-ethereum/blob/295bc35ecf9b29f72db8d71f61e2d65b4d4bb24c/signer/core/signed_data.go#L129

Reply:
We might need to keep the Sign method as part of our store BUT we don't expose it in our API

Reply:
done